src/vss_tools/vspec/utils/idgen_utils.py

```diff
@@ -13,8 +14,8 @@ def get_node_identifier_bytes(
     node_type: str,
     unit: str,
     allowed: str,
-    minimum: str,
-    maximum: str,
+    minimum: Optional[str],
+    maximum: Optional[str],
```

Comment:
same here it should probably be `minimum: int | float | None` and same for maximum, no?

Reply:
Done